src/spikeinterface/widgets/unit_waveforms.py

```diff
@@ -274,8 +352,9 @@ def plot_ipywidgets(self, data_plot, **backend_kwargs):
                 self.fig_probe, self.ax_probe = plt.subplots(figsize=((ratios[2] * width_cm) * cm, height_cm * cm))
                 plt.show()
 
-        self.unit_selector = UnitSelector(data_plot["unit_ids"])
+        self.unit_selector = UnitSelector(data_plot["unit_ids"], layout=widgets.Layout(height="80%"))
         self.unit_selector.value = list(data_plot["unit_ids"])[:1]
+        self.scaler = ScaleWidget(layout=widgets.Layout(height="20%"))
```

Comment:
Maybe we could expose the initial value of the sclae (default 1) also for matpltolib

Reply:
added it for matplotlib and ipywidgets. Moved the scaling mechanism to `plot_matplotlib`